src/main/java/org/reservationapplication/service/CacheServiceCoworkingSpace.java

```diff
@@ -0,0 +1,40 @@
+package org.reservationapplication.service;
+
+import com.github.benmanes.caffeine.cache.Cache;
+import com.github.benmanes.caffeine.cache.Caffeine;
+import org.reservationapplication.model.CoworkingSpace;
+import org.reservationapplication.repository.CoworkingSpaceRepository;
+
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+public class CacheServiceCoworkingSpace {
+    private final CoworkingSpaceRepository repository = new CoworkingSpaceRepository();
+    private final Cache<String, List<CoworkingSpace>> cache;
+
+    public CacheServiceCoworkingSpace() {
+        this.cache = Caffeine.newBuilder()
```

Comment:
damn, i left the comment in repo, why here? but thumb up for caffeine. For now it would be better to implement it yourself but ok